src/Maui/Prism.Maui/Navigation/INavigationService.cs

```diff
@@ -20,6 +20,21 @@ public interface INavigationService
     /// <returns>If <c>true</c> a go back operation was successful. If <c>false</c> the go back operation failed.</returns>
     Task<INavigationResult> GoBackAsync(INavigationParameters parameters);
 
+    /// <summary>
+    /// Navigates to the most recent entry in the back navigation history for the <paramref name="viewName"/>.
+    /// </summary>
+    /// <param name="viewName">The name of the View to navigate back to</param>
+    /// <returns>If <c>true</c> a go back operation was successful. If <c>false</c> the go back operation failed.</returns>
+    Task<INavigationResult> GoBackAsync(string viewName);
```

Comment:
This should be an extension on
```cs
Task<INavigationResult> GoBackAsync(string viewName, INavigationParameters parameters)
```
where we just pass in empty NavigationParameters.

Reply:
Move to extension method  
https://github.com/PrismLibrary/Prism/pull/3048/files#diff-df76ce41111618523c5ba67f2a5750376a834eba0a35dc12babad363edac96faR42